helmclient.go

```diff
@@ -583,7 +585,7 @@ func getPodName(client kubernetes.Interface, labelSelector, namespace string) (s
 		return "", microerror.Maskf(tooManyResultsError, "%d", len(pods.Items))
 	}
 	if len(pods.Items) == 0 {
-		return "", microerror.Maskf(podNotFoundError, "%s", labelSelector)
+		return "", microerror.Maskf(notFoundError, "%s", labelSelector)
```

Comment:
`tooManyResultsError` error was generic one. This was specific. I think this method is so well scoped that returning generic error makes more sense here.